content/doc/book/security/access-control/permissions.adoc

```diff
@@ -107,6 +105,95 @@ Anything they cannot accomplish through the existing UI they can do through the
 An instance on which an untrusted user gained _Overall/Administer_ permission should be considered fully compromised and should be replaced.
 All secrets (credentials, etc.) stored on that instance should be rotated or revoked and all artifacts created from that point on should be verified.
 
+
+=== Agent permissions ===
+
+==== Agent/Build ====
+
+This permission allows users to run jobs as them on agents.
+
+==== Agent/Configure
+
+This permission allows users to configure agents.
+
+==== Agent/Connect
+
+This permission allows users to connect agents or mark agents as online.<br/><br/>This permission is implied by Agent/Disconnect.
+
+==== Agent/Create
+
+This permission allows users to create agents.
+
+==== Agent/Delete
+
+This permission allows users to delete existing agents.
+
+==== Agent/Disconnect
+
+This permission allows users to disconnect agents or mark agents as temporarily offline.
+
+=== Job permissions
+
+==== Job/Build
+
+This permission grants the ability to start a new build.
+
+==== Job/Cancel
+
+This permission grants the ability to cancel a scheduled, or abort a running, build.
+
+==== Job/Configure
+
+Change the configuration of a job.
+
+==== Job/Create
+
+Create a new job.
+
+==== Job/Delete
+
+Delete a job.
+
+==== Job/Discover
+
+This permission grants discover access to jobs. Lower than read permissions, it allows you to redirect anonymous users to the login page when they try to access a job url. Without it they would get a 404 error and wouldn't be able to discover project names.<br/><br/>This permission is implied by Job/Read.
```

Comment:
Preference is for sentence per line, though that preference is not strongly enforced.

Simplify the markup for easier reading
```suggestion
This permission grants discover access to jobs. Lower than read permissions, it allows you to redirect anonymous users to the login page when they try to access a job url.
Without it they would get a 404 error and wouldn't be able to discover project names.

This permission is implied by Job/Read.
```